app/_data/docs_nav_2.1.x.yml

```diff
@@ -452,6 +452,8 @@
           url: /../../kubernetes-ingress-controller/latest/concepts/ingress-classes
         - text: Security
           url: /../../kubernetes-ingress-controller/latest/concepts/security
+        - text: Ingress Versions
```

Comment:
Not clear what "ingress versions" refers to. For example, is this versions of an ingress resource (if you apply consecutive patches to an ingress X in namespace Y) or API versions of the Ingress kind?

I recommend more specific alternatives:
- `networking.k8s.io` API versions (or `networking.k8s.io` versions)
- Ingress Resource API versions